web-services-app/src/main/resources/search.properties

```diff
@@ -4,11 +4,12 @@ search.results.pagesToDisplay=10
 search.results.neighborItems=7
 search.results.maxBrowsePerPage=100
 #Fields which are searchable via keyword searches
-search.field.searchable=ID,DEFAULT_INDEX,TITLE_INDEX,CONTRIBUTOR_INDEX,SUBJECT_INDEX,IDENTIFIER\
+search.field.searchable=ID,DEFAULT_INDEX,TITLE_INDEX,CREATOR_INDEX,CONTRIBUTOR_INDEX,SUBJECT_INDEX,IDENTIFIER\
 ,RLA_SITE_CODE,RLA_CATALOG_NUMBER,RLA_CONTEXT_1
 #Set to one higher than you want to display as the last value indicates whether the UI should show a "show more" link
 search.facet.facetsPerGroup=6
 search.facet.expandedFacetsPerGroup=20
 search.facet.maxFacetsPerGroup=50
-search.facet.fields=ANCESTOR_PATH,CONTENT_TYPE,DEPARTMENT,DEPARTMENT_LC,SUBJECT,GENRE,LANGUAGE,LOCATION,PARENT_COLLECTION,CONTENT_MODEL,RESOURCE_TYPE,STATUS,CONTENT_STATUS,SCOPE,ROLE_GROUP
+search.facet.fields=ANCESTOR_PATH,CONTENT_TYPE,DEPARTMENT,DEPARTMENT_LC,SUBJECT,GENRE,LANGUAGE,LOCATION,\
+  PARENT_COLLECTION,CONTENT_MODEL,RESOURCE_TYPE,STATUS,CONTENT_STATUS,SCOPE,ROLE_GROUP,CREATOR,CONTRIBUTOR
```

Comment:
I'd suggest also adding `CREATOR_CONTRIBUTOR` here, so the api can work with creator, contributor, and both of them combined (since the other apps can do that last one)